packages/sdk/src/types/network/payloads/payloads.ts

```diff
@@ -317,6 +317,17 @@ export type SetBehavior = Payload & {
     behaviorType: BehaviorType;
 };
 
+/**
+ * @hidden
+ * App to engine.  Update whether the actor is grabbable
+ * or not.
+ */
+export type UpdateGrabbable = Payload & {
+    type: 'update-grabbable';
+    actorId: string;
+    grabbable: boolean;
+};
+

```

Comment:
In thinking about this some more, I think that this will not be needed, as the grabbable state should be able to be updated through patching alone, but will investigate this further in the Unity runtime to be sure. Will likely be removed.

Reply:
Yep I think this should be done via actor patch.

---
In reply to: [261902987](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/233#discussion_r261902987) [](ancestors = 261902987)